extensions/wikia/DefaultMessages/DefaultMessages.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+$wgExtensionCredits[ 'other' ][ ] = array(
+	'name' => 'DefaultMessages',
+	'author' => 'Wikia',
+	'descriptionmsg' => 'defaultmessages-desc',
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/DefaultMessages',
+);
+
+//i18n
+$wgExtensionMessagesFiles['DefaultMessages'] = __DIR__ . '/DefaultMessages.i18n.php';
+
+$wgExtensionFunctions[] = 'efDefaultMessagesSetup';
+$wgAutoloadClasses['DefaultMessagesCache'] = dirname(__FILE__) . '/DefaultMessagesCache.php';
+
+function efDefaultMessagesSetup() {
+	global $wgDefaultMessagesDB, $wgDefaultMessagesCache;
+	global $wgDBname, $wgHooks;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgHooks" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)